### DISCUSSION TITLE
Question about GUI Availability and Usage Instructions

### DISCUSSION THREAD

**Discussion by puhlenkiyroman** [mehdihadeli/go-food-delivery-microservices#111] [Q&A]
Hello, dear Mehdi Hadeli.

We are using this project at university pracitve for infosec, testing SonarQube, pentesting and other.
I would like to inquire if there is a graphical user interface (GUI) available in this project. If so, could you please provide some guidance on how to access and use it?

Thank you for your help!

Best regards, Roman Puhlenkov

**Comment by mehdihadeli** [comment]
Hi,
No UI currently, maybe in the future.

**Comment by puhlenkiyroman** [comment]
Can you explain the main idea of app? How we can use it? And can you suggest what we as infosec can do with this project ?

**Comment by mehdihadeli** [comment]
This is a microservices architecture sample in golang just for practicing different technologies in a project, if someone need to implement a microservices based application they can get some idea from this project. Maybe it is not good for your usage.
